scripts/end_to_end_demo.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+sudo snap install microk8s --classic --channel=1.18/stable
+
+sudo usermod -a -G microk8s dazwilkin
+
+sudo microk8s status --wait-ready
+
+sudo microk8s enable dns helm3 rbac
+
+echo "--allow-privileged=true" >> /var/snap/microk8s/current/args/kube-apiserver
```

Comment:
Maybe change to the following to avoid permissions error:
```
echo "--allow-privileged=true" | sudo tee -a /var/snap/microk8s/current/args/kube-apiserver
```

Reply:
For some context: When I ran it with the original line, the agent pods wouldn't be deployed because allow privileged was never set due to permissions errors.